db/migrate/20230617125650_rename_author_id_to_user_id_in_likes_and_comments.rb

```diff
@@ -0,0 +1,10 @@
+class RenameAuthorIdToUserIdInLikesAndComments < ActiveRecord::Migration[7.0]
+  def change
+    add_index :posts, :author_id, unique: true
+    add_foreign_key :likes, :users, column: :user_id
+    add_foreign_key :likes, :posts, column: :post_id
+
+    add_foreign_key :comments, :users, column: :user_id
+    add_foreign_key :comments, :posts, column: :post_id
+  end
+end
```

Comment:
- [ ] Hello there, @Odongo006! You've done well so far in going over the project requirements 👏. However, I am unable to run this migration due to the following error: **relation "index_posts_on_author_id" already exists**. Please ensure that this is corrected and that all migrations are running properly. Before submitting for another review, please ensure that all of the associations between the models are correctly set up according to the provided [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png). :ok_hand: 😊

### Error running the migration :point_down: 

![Screenshot from 2023-06-17 14-32-21](https://github.com/Odongo006/My-Blog-App/assets/89228743/ea80c9eb-5a76-46d9-a82e-5558be2f3c44)